slicerator.py

```diff
@@ -493,9 +506,35 @@ def pipeline(func):
 
     The function can still be applied to ordinary images. The decorator
     only takes affect when a Slicerator object is passed.
+
     >>> single_img = images[0]
     >>> red_img = red_channel(single_img)  # normal behavior
     """
+    def wrapper(f):
+        return _pipeline(f, kwargs.get("retain_doc", False))
```

Comment:
Why not pass all `**kwargs` over here? Now typos like `retin_doc` will be silently accepted.